netbox-event-driven-ansible/rulebooks/aws-sqs-rulebook-template.yml

```diff
@@ -0,0 +1,23 @@
+
+- name: Events on SQS
+  sources:
+    - ansible.eda.aws_sqs_queue:
+        access_key: "{{ aws_access_key }}"
```

Comment:
Is it using Event Streams or webhooks?

Reply:
good point this is placeholder code for SQS, I will remove it